www/docs/tutorials/securing-pages-and-api-routes.md

```diff
@@ -69,7 +69,7 @@ export async function getServerSideProps(context) {
 This example assumes you have configured `_app.js` to pass the `session` prop through so that it's immediately available on page load to `useSession`.
 
 ```js title="pages/_app.js"
-import { Provider } from 'next-auth/client'
+import { Provider } from "next-auth/react"
```

Comment:
Looks like you've renamed `Provider` to `SessionProvider` so all documentation references need to be updated as well.

Reply:
Exactly. This is still a WIP, but thanks for the heads up. :)